a1/README.md

```diff
@@ -406,7 +406,9 @@ While this guide explains the *what*, *why* and *how*, I find it helpful to see
   Note: When creating watches in a controller using `controller as`, you can watch the `vm.*` member using the following syntax. (Create watches with caution as they add more load to the digest cycle.)
 
   ```html
-  <input ng-model="vm.title"/>
+  <div ng-controller="SomeController as vm">
```

Comment:
I don't think this is a good addition. There are many ways to use `controllerAs` (e.g. directives, routing) and `ngController` is not considered a good practice any more.

(Also note that this is a styleguide, not a tutorial, so people are expected to know one thing or two about Angular.)